bot/skills/__init__.py

```diff
@@ -34,16 +38,61 @@
 logger = logging.getLogger(__name__)
 
 
+class ChatCommandHandler(CommandHandler):
+    """ChatCommandHandler is class-wrapper for `CommandHandler`. It provides default `chat_id` filtering.
+    `chat_id` takes from `config.get_group_chat_id() -> str` and it could be either
+    chat_username :: str (for public groups) or chat_id :: int (for private or public). So, this wrapper
+    support both type of `chat_id`, adds chat_id filtering and set `run_async` to True by default.
+    """
+
+    def __init__(
+        self,
+        command: SLT[str],
+        callback: Callable[[Update, CCT], RT],
+        filters: BaseFilter = None,
+        allow_edited: bool = None,
+        pass_args: bool = False,
+        pass_update_queue: bool = False,
+        pass_job_queue: bool = False,
+        pass_user_data: bool = False,
+        pass_chat_data: bool = False,
+        run_async: Union[bool, DefaultValue] = DEFAULT_FALSE,
```

Comment:
Why this is better than `bool = False`?

Reply:
I suggest you to apply Ockhams Rasiermesser here. Generally, an answer is pretty clear: I just override parent's constructor. And `run_async: Union[bool, DefaultValue] = DEFAULT_FALSE` is just a part of it.